src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallsControllerTest.kt

```diff
@@ -216,4 +248,56 @@ class RecallsControllerTest {
 
     assertThat(result, equalTo(RecallResponse(recallId, nomsNumber)))
   }
+
+  private val recall = Recall(recallId, nomsNumber)
```

Comment:
I'd rather we didn't paste class variables into the middle of the class but 🤷‍♂️